myapp/deleteClient.js

```diff
@@ -0,0 +1,21 @@
+const request = require('request');
+const app = require('./app.js');
+
+deleteClient = function () { 
+  app.delete('/clients/:client', function (req, res) {
```

Comment:
- According to the REST which we covered earlier, should this be `/clients/:clientId`?
- Use arrow function in your callback